### PR TITLE
Js classes functionality

### PR DESCRIPTION
**Pull request by serengia**
Request to merge js-classes branch to main.
This branch contains the following features.
- Javascript class implementation
- Booklist and form styling

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,76 +1,98 @@
...
+
+      //   Clear inputs
+      e.target.title.value = "";
+      e.target.author.value = "";
+
+      //   Add book
+      this.books.push(formData);
+
+      //   Save to local storage
+      //   console.log(JSON.stringify(books));
```
Comment (leonardodiasb) [inline script.js]: - [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy.

File: script.js [hunk 2]
```diff
@@ -1,76 +1,98 @@
...
+    this.booksListContainer.addEventListener("click", (e) => {
+      const clickedButton = e.target.closest(".btn-remove");
+
+      if (!clickedButton) return;
+
+      const { id } = clickedButton.dataset;
+
+      this.books = this.books.filter((book) => book.id !== id);
+
+      //   console.log(this);
```
Comment (leonardodiasb) [inline script.js]: - [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Serengia, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Correct use of GitHub flow. ✔️
 - No linter errors. ✔️ 
 - Professional PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @Serengia, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**